libraries/AP_Proximity/AP_Proximity_Backend.h

```diff
@@ -36,6 +37,9 @@ class AP_Proximity_Backend
     // update the state structure
     virtual void update() = 0;
 
+    // This method helps frontend access 3D boundary methods
```

Comment:
This comment should probably be changed to perhaps, "timeout faces that have not received data recently and update filter frequencies"